cfg/config.go

```diff
@@ -3,11 +3,98 @@
 // that will be retrieved from  .env or ./cfg/config.yml
 package cfg
 
+import (
+	"bufio"
+	"log"
+	"os"
+	"strings"
+	"time"
+)
+
+// Load configs from a env file & sets them in environment variables .
+func LoadEnvVar() error {
+
+	f, err := os.Open(".env")
+
+	if err != nil {
+		log.Printf("%s", err)
+		return err
+	}
+
+	defer func() {
+		err := f.Close()
+		if err != nil {
+			log.Printf("%s", err)
+		}
+	}()
+
+	lines := make([]string, 0)
+	scanner := bufio.NewScanner(f)
+
+	for scanner.Scan() {
+		lines = append(lines, scanner.Text())
+	}
+
+	if err := scanner.Err(); err != nil {
+		log.Printf("%s", err)
+		return err
+	}
+
+	for _, l := range lines {
+		pair := strings.Split(l, "=")
+		os.Setenv(pair[0], pair[1])
+	}
+	return nil
+}
+
+var (
```

Comment:
Are you sure we have to use these vars here? We could simply write values into a struct fields

Reply:
We could read config from the environment after setting up variables.  I am not sure it is a good approach when it comes to testing, but it allows us to decouple our layers from cfg package and will simplify our code a little.

Reply:
> Are you sure we have to use these vars here? We could simply write values into a struct fields

Yes, you're right. I made a change.

Reply:
> We could read config from the environment after setting up variables. I am not sure if it is a good approach when it comes to testing, but it allows us to decouple our layers from cfg package and will simplify our code a little.

I think I need help here. I'm performing a ritual to summon wise spirits: @vpakhuchyi, @andstepko, @Kalachevskyi 
Somebody give a good omen)

Reply:
I think it's fine for now. We always can change it once ti's needed.